index.html

```diff
@@ -80,6 +83,72 @@ <h1 class="main-description">
           </li>
         </ul>
     </section>
+    <section class="project">
+    <div class="card">
+      <img class="pic" src="img/Snapshoot Portfolio.png" alt="First-img" />
+      <p class="card-name">Tonic</p>
+      <ul class="can-bak">
+        <li class="cann">canopy</li>
+        <li class="bak">Back End Dev</li>
+        <li class="bak">2015</li>
+      </ul>
+      <p class="tex-sec">A daily selection of privately personalized read;no account or signs-up required</p>
+      <ul class="work-list">
+      <li class="work-name">html</li>
+        <li class="work-name">css</li>
+        <li class="work-name">javascript</li>
+      </ul>
+      <button class="sec-button">see project</button>
```

Comment:
The design of your button looks great, and you can add some effects to improve it. For example, the hover, active, and disabled states, as you can see in the below screenshot from Figma.

![image](https://user-images.githubusercontent.com/34779928/192015264-3039d795-be7c-411f-8cfd-f55f7080f56e.png)